test/e2e/tests/tokens/custom-token-send-transfer.spec.ts

```diff
@@ -61,7 +61,7 @@ describe('Transfer custom tokens', function () {
           );
 
           // edit gas fee
-          await tokenTransferRedesignedConfirmPage.editGasFee(
```

Comment:
renamed, because this edit gas tests are using a legacy hardfork, so the edit gas  legacyis specific for that case.